### PR TITLE
Render visuazliations from @tiledb-inc/viz-core

### PR DESCRIPTION
**Pull request by SarantopoulosKon**
Run `jlpm` or `yarn` to install new dependencies before testing

### CODE REVIEW

File: src/widget.ts [hunk 1]
```diff
@@ -70,54 +39,15 @@ abstract class BabylonBaseView extends DOMWidgetView {
   inspector = this.values.inspector;
 
   protected resizeCanvas(): void {
```
Comment (alekshs) [inline src/widget.ts:41]: Is this method redundant? It also exists in core package and is called upon render.

File: src/widget.ts [hunk 2]
```diff
@@ -60,7 +29,7 @@ export class BabylonBaseModel extends DOMWidgetModel {
 
 abstract class BabylonBaseView extends DOMWidgetView {
   canvas?: HTMLCanvasElement;
-  engine?: Engine;
+  Visualization?: TileDBVisualization;
```
Comment (alekshs) [inline src/widget.ts:32]: Maybe lower case the **instance** name?

File: src/widget.ts (hunk 1 above)
Comment (SarantopoulosKon) [reply src/widget.ts:41]: Tbh i don't see anywhere calling it, maybe @MargrietGroenendijk can answer about this.

Comment (alekshs) [reply src/widget.ts:41]: It has been moved to the core package

**Review by MargrietGroenendijk — COMMENTED** [review]
Running some tests with  a local notebook: 
* the default point size has changed as the points are too large now. Can you set this to 1.0? I think [here](https://github.com/TileDB-Inc/TileDB-Viz/blob/97da0f6a160ec03c9385638fed9a9921e67c9be4/packages/core/src/point-cloud/point-cloud.ts#L158)? Changing the value of `point_size` in the Python API does not change it either
* The legacy class `BabylonJS` in show.py  (used in the Boulder cloud demos) does not work and gives this error: `error.js:11 Uncaught (in promise) RuntimeError: Unable to load from blob:***: Cannot read properties of null (reading 'asset')`
* [Clash detection is not working](https://github.com/TileDB-Inc/TileDB-Viz/blob/97da0f6a160ec03c9385638fed9a9921e67c9be4/packages/core/src/point-cloud/point-cloud.ts#L533) - after selecting a mesh nothing happens with `=`
* Error when `isClass` is true: `babylon.gui.min.js:16 Uncaught TypeError: Cannot read properties of undefined (reading 'split')`
* Error when `isTopo` is true: the mapbox layer is not displayed
* When `isGltf` is true the variables `mesh_shift`, `mesh_rotation` and `mesh_scale` do not move and scale the mesh

**Comment by alekshs** [comment]
> Running some tests with a local notebook:
> 
> * the default point size has changed as the points are too large now. Can you set this to 1.0? I think [here](https://github.com/TileDB-Inc/TileDB-Viz/blob/97da0f6a160ec03c9385638fed9a9921e67c9be4/packages/core/src/point-cloud/point-cloud.ts#L158)? Changing the value of `point_size` in the Python API does not change it either
> * The legacy class `BabylonJS` in show.py  (used in the Boulder cloud demos) does not work and gives this error: `error.js:11 Uncaught (in promise) RuntimeError: Unable to load from blob:***: Cannot read properties of null (reading 'asset')`
> * [Clash detection is not working](https://github.com/TileDB-Inc/TileDB-Viz/blob/97da0f6a160ec03c9385638fed9a9921e67c9be4/packages/core/src/point-cloud/point-cloud.ts#L533) - after selecting a mesh nothing happens with `=`
> * Error when `isClass` is true: `babylon.gui.min.js:16 Uncaught TypeError: Cannot read properties of undefined (reading 'split')`
> * Error when `isTopo` is true: the mapbox layer is not displayed
> * When `isGltf` is true the variables `mesh_shift`, `mesh_rotation` and `mesh_scale` do not move and scale the mesh

The default pointSize has been fixed [here](https://github.com/TileDB-Inc/TileDB-Viz/pull/20/commits/64e38ae68f71aaf650841bda05e9bf2f487d6109)
Also, the properties passed to the JS package have been converted to camel case, maybe that's why your config does not apply.

**Review by MargrietGroenendijk — APPROVED** [review]

**Pull request merged**